### DISCUSSION TITLE
Enhancement request: clean (vanilla) runners on Github Actions

### DISCUSSION THREAD

**Discussion by jmarrec** [community/community#38056] [Actions]
It would be terrific if we could also choose a Github-hosted runner for each platform that does not come preinstalled with any developer tools, especially for testing installers.

For Linux, I can run on ubuntu docker container, but for macOS and Windows there are no alternatives.

Context: I keep getting burned by having my installers test correctly on the windows-latest / macos-latest but in reality it's picking up developer tools & libraries that are preinstalled.

**Comment by david-engelmann** [comment]
An "alpine" windows and "alpine" macos runners would be pretty sweet

**Comment by ossanna16** [comment]
Hi there @jmarrec and welcome to our community! Thank you for the feedback 🙂
